prow/kube/client.go

```diff
@@ -324,8 +332,7 @@ func (c *Client) GetPod(name string) (Pod, error) {
 	c.log("GetPod", name)
 	var retPod Pod
 	err := c.request(&request{
-		method: http.MethodGet,
```

Comment:
Why did you remove `method: http.MethodGet` throughout? Does the method default to GET somewhere?

Reply:
Yes, that's the default in go's http package.